go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -165,6 +167,7 @@ func getSvcInfo(coreclient *corev1client.CoreV1Client, restconfig *rest.Config,
 	svcInfo = &SvcInfo{
 		IP: svc.Spec.ClusterIP,
 	}
+	svcInfo.IsIPv4 = strings.Count(svcInfo.IP, ":") < 2
```

Comment:
@mrVectorz canonical way to do this would be to flip the IsIPv4 -> IsIPv6 and then:

```
if net.ParseIP(svcInfo.IP).To4() == nil {
    svcInfo.IsIPv6 = true
}
```

Reply:
But see below...